### PR TITLE
Brings the scrollable columns into the enterprise release

### PR DESCRIPTION
**Pull request by rauhryan**
- Various css fixes specific to safari issues in the milestone layout #312
  - Chains the card extension to fix failed payload merges #311
  - Make the columns scrollable with horizontal scroll on the milestones view #280
  - Consistent Word Break Handling #240
  - Accommodates custom media types #306
  - Adds a composable client-side parser in prep for Card Relationships #302
  - Prevents QP's from polluting others with matching name #316
  - Adds deployed boilerplate for PR timelines #320

### CODE REVIEW

**Comment by dahlbyk** [comment]
> Consistent Word Break Handling #240

For the record, this has been in `enterprise` for quite a while.

**Comment by rauhryan** [comment]
Yeah, I thought so too, it showed up in the diff somehow.

File: Gemfile.lock [hunk 1]
```diff
@@ -23,6 +17,11 @@ GIT
       warden (> 1.0)
 
 PATH
+  remote: vendor/engines/enterprise
+  specs:
+    enterprise (0.0.1)
+
```
Comment (dahlbyk) [inline Gemfile.lock]: Should this engine have a rails dependency?

File: Gemfile.lock [hunk 2]
```diff
@@ -539,3 +538,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   warden-github (= 1.4.0)!
   wkhtmltopdf-heroku
+
+BUNDLED WITH
+   1.11.2
```
Comment (dahlbyk) [inline Gemfile.lock]: This had been specifically removed to avoid bundler version mismatches - is it cool now?

Comment (rauhryan) [inline Gemfile.lock]: I guess so, seems to be working fine

**Pull request merged**